Engine/Settings.cs

```diff
@@ -477,21 +576,19 @@ private void parseSettingsFile(string settingsFilePath)
                 // it is not available on PSv3, we resort to our own narrow implementation.
                 hashtable = GetSafeValueFromHashtableAst(hashTableAst);
             }
-            catch (InvalidOperationException e)
+            catch (InvalidOperationException exception)
             {
-                throw new ArgumentException(Strings.InvalidProfile, e);
+                throw new ArgumentException(Strings.InvalidProfile, exception);
             }
 
-            if (hashtable == null)
+            if (hashtable is null)
```

Comment:
`==` is more appropriate here since it really does test referential equality. The only language I know where `is` is conventional is Python (because it doesn't really have a `null`, but a singleton object `None` that references default to).